app/models/work.rb

```diff
@@ -189,6 +189,7 @@ def check_for_invalid_chapters
   after_update :adjust_series_restriction
 
   after_save :expire_caches
+  after_destroy :expire_caches
```

Comment:
There's a test for this we can now re-enable: https://github.com/otwcode/otwarchive/blob/218f15e009438e920c8eef8ae651ba51c5d05edf/spec/lib/works_owner_spec.rb#L48